scripts/minishift-all-tests.sh

```diff
@@ -0,0 +1,47 @@
+#!/usr/bin/env bash
+
+executing() {
+   set +x
+  echo -e "\n------------------------------\n${1}\n------------------------------\n"
+   set -x
+}
+
+set -ex
+
+#Export GitHub token to avoid 
+export MINISHIFT_GITHUB_API_TOKEN="03d362b0f2a7ecf02b30bfbdd40fe720a4d6b8b3"
```

Comment:
This should not be there. It is a big security hole! 
It needs to use some kind of mechanism for securing secrets.

Do we even need this? If I remember correctly this token is only used by minishift to check for new versions and it is not required for any other functionality.